src/tiler/src/api.js

```diff
@@ -85,8 +86,9 @@ api.get(
             const { z, x, y } = processCoords(req)
             const utfFields = processUTFQuery(req)
             const layers = processLayers(req)
+            const { config } = req.queryString
```

Comment:
Now that there are three things repeated three times I'm tempted to call for them to get bundled up in another helper function, but that's probably excessive.  Especially since there's also a 4th thing here, so clearly there's some variation in what's needed.  Though speaking of which, if the three common lines do stay repeated, it might be nice to move the `utfFields` line to the end to make it clearer what's the same and what's different between the methods.